config/env.yml.template

```diff
@@ -0,0 +1,11 @@
+defaults: &defaults
+  SECRET_KEY_BASE: 'secret_key_base'
+  OMNIAUTH_CLIENT_SECRET: 'omniauth_client_secret'
+  OMNIAUTH_CLIENT_ID: 'omniauth_client_id'
+  DEVISE_SECRET_KEY: 'devise_secret_key'
+
+development:
+  <<: *defaults
+
+test:
+  <<: *defaults
```

Comment:
@EricBoersma I'm now using envyable in development and test. It'll seed END on initilaisation.  This PR comes with a env.yml.template, and the README has been updated.